apps/remix-ide/src/app/ui/confirmDialog.js

```diff
@@ -25,24 +25,48 @@ function confirmDialog (tx, amount, gasEstimation, self, newGasPriceCb, initialP
   }
 
   var el = yo`
-  <div>
-    <div>You are creating a transaction on the main network. Click confirm if you are sure to continue.</div>
-    <div class=${css.txInfoBox}>
-      <div>From: ${tx.from}</div>
-      <div>To: ${tx.to ? tx.to : '(Contract Creation)'}</div>
-      <div>Amount: ${amount} Ether</div>
-      <div>Gas estimation: ${gasEstimation}</div>
-      <div>Gas limit: ${tx.gas}</div>
-      <div>Gas price: <input id='gasprice' oninput=${onGasPriceChange} /> Gwei <span> (visit <a target='_blank' href='https://ethgasstation.info'>ethgasstation.info</a> to get more info about gas price)</span></div>
-      <div>Max transaction fee:<span id='txfee'></span></div>
-      <div>Data:</div>
-      <pre class=${css.wrapword}>${tx.data && tx.data.length > 50 ? tx.data.substring(0, 49) + '...' : tx.data} ${copyToClipboard(() => { return tx.data })}</pre>
+    <div>
+      <div>You are about to create a transaction on the main network. Confirm the details to send to your provider.</div>
```

Comment:
For many users, I think the word **provider** is not clear.  Is this line better: 

You are about to create a transaction on the main network. Confirm the details to send to your provider.  The provider for many users is MetaMask.  Confirming the details only sends the info to the provider -where in turn, you will need to sign the transaction to send it to the main network.

or 

You are about to create a transaction on the main network. Confirm the details to send the info to your provider.  The provider for many users is MetaMask.  The provider will ask you to sign the transaction before it is sent to the main network.